cdc/processor.go

```diff
@@ -245,6 +245,15 @@ func newProcessor(
 	p.status = status
 	p.statusModRevision = modRevision
 
+	info, _, err := p.etcdCli.GetChangeFeedStatus(ctx, p.changefeedID)
+	if err != nil && cerror.ErrChangeFeedNotExists.NotEqual(err) {
+		return nil, errors.Trace(err)
+	}
+
+	if err == nil {
+		p.globalcheckpointTs = info.CheckpointTs
+	}
```

Comment:
Is it dangerous if `p.globalcheckpointTs` is not initialized here? Such as will the L988 check fail?

```
if replicaInfo.StartTs < globalcheckpointTs {
    log.Panic("addTable: startTs < checkpoint",
}
```